_pages/story.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: default
-title: History, present, and future
+title: History and press
 ---
 
-# History, present, and future
+# History and press
 
-We would like other agencies, developers, and groups to use eRegulations and adapt it. Here's some insight into the current status of the project and where it's going.
+The CFPB released its initial version of eRegulations [in 2013](http://www.consumerfinance.gov/blog/making-regulations-easier-to-use/), and 18F and ATF began reusing it for ATF [in 2015](https://18f.gsa.gov/2015/12/09/an-open-source-government-is-a-faster-more-efficient-government/).
```

Comment:
Ah yeh, I noticed that but wasn't sure if I should edit it. I simplified it a bit.